### PR TITLE
V0.2

### PR DESCRIPTION
**Pull request by wesmith4**

### CODE REVIEW

File: lib/stringToChars.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+function stringToChars(string) {
+  let arrayChars = [];
+  for (let i of string) {
+    arrayChars.push(i);
+  }
+  return (arrayChars);
+}
+stringToChars('Hello');
```
Comment (jfarmer) [inline lib/stringToChars.js:9]: This line will be run every time you `require` this file. You'll want to remove it.

File: lib/stringToChars.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+function stringToChars(string) {
+  let arrayChars = [];
```
Comment (jfarmer) [inline lib/stringToChars.js:3]: You could also do:

```javascript
function stringToChars(array) {
  return string.split('');
}
```

File: lib/stringToChars.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  for (let i of string) {
```
Comment (jfarmer) [inline lib/stringToChars.js:4]: Be careful with a variable name like `i` here. 99% of programmers will believe a variable named `i` is meant to be an index into a list.

Something like:

```javascript
for (let char of string) {
  // ..
}
```

is clearer.

File: lib/stringToChars.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+
+function stringToChars(string){
+    let arrayChars = [];
+    for (let i of string){
```
Comment (HuseyinAltnsk) [inline lib/stringToChars.js]: You can also do `string.split('')`. The `.split()` function would split `cat` and return the array `['c', 't']`, but if you pass `''` (empty string) as argument, it splits by empty strings, which is on the left/right of every letter, so you get an array of all the characters other than the empty string, which is `['c', 'a', 't']`

**Pull request merged**